app/qml/components/MMMapBlurLabel.qml

```diff
@@ -0,0 +1,58 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import Qt5Compat.GraphicalEffects
+import "."
+
+Item {
+  id: root
+
+  property alias sourceItem: effect.sourceItem
+  property alias text: text.text
+
+  Rectangle {
+    anchors.fill: fastBlur
+    color: __style.forestColor
+    opacity: 0.8
+    radius: 21
+  }
+
+  FastBlur {
+    id: fastBlur
+
+    x: 20 * __dp
```

Comment:
It will be tricky to use it this way. I would say let's start from `0` and let the item that creates this object set margins instead (like with other map elements).

Is there any reason to set it up here?